test-cases/gutenberg/columns.md

```diff
@@ -7,6 +7,7 @@
 ### Columns block is added with default number of Column
 
 - Add a `Columns` block
+- Press `Cancel` button on layout picker
```

Comment:
Is pressing cancel critical for these cases? If selecting one of the layouts don't effect the tests we don't need to state it. It is better if we can keep steps simple, needs less maintenance that way.

Reply:
It's not critical, I just wanted to be detailed in steps. We can remove it, it shouldn't affect the next steps.